tslint.ts

```diff
@@ -155,6 +155,9 @@ const rules : tslint.Configuration.RawRulesConfig = Object.assign({}, rulesAirbn
     groups: importGroups,
   }],
 
+  // for the tslint-react-hooks plugin
+  'react-hooks-nesting': true
```

Comment:
but isn't trailing comma missing now?

Reply:
Had no problems without it, but added it now.